spec/graphql/mutations/entitlement/destroy_feature_spec.rb

```diff
@@ -3,15 +3,13 @@
 require "rails_helper"
 
 RSpec.describe Mutations::Entitlement::DestroyFeature, type: :graphql do
-  let(:required_permission) { "features:delete" }
-  let(:membership) { create(:membership) }
-  let(:organization) { membership.organization }
+  include_context "with graphql query context"
```

Comment:
I considered adding this automatically when `type: :graphql` is present, via spec_helper.rb but I thought it was not explicit enough.

```rb
RSpec.configure do |config|
  # ...
  config.include_context "with graphql query context", type: :graphql
  # ...
end
```